app/index.html

```diff
@@ -25,9 +24,8 @@
 	<div id="Radio4000" class="Radio4000"></div>
 	<noscript>Sorry, to use Radio4000 you need to enable JavaScript in your browser.</noscript>
 
-	<!-- <script src="{{rootURL}}assets/vendor.js"></script> -->
-	<!-- <script src="{{rootURL}}assets/radio4000.js"></script> -->
-	{{content-for "concat-js"}}
+	<script src="{{rootURL}}assets/vendor.js"></script>
+	<script src="{{rootURL}}assets/radio4000.js"></script>
```

Comment:
mind keeping this one as `async`?

Reply:
why keeping?
just the one on line 28?

Reply:
Yes, just the second. So we're sure it loads after the first.

It should be faster when async, so it doesn't block rendering.